CONTRIBUTING.md

```diff
@@ -57,11 +57,11 @@ is [purposefully excluded from the project's `Gemfile`][exclude].
 [foreman]: https://github.com/ddollar/foreman
 [exclude]: https://github.com/ddollar/foreman/pull/437#issuecomment-41110407
 
-The application will run locally at `http://localhost:5000/`.
+The application will run locally at `http://localhost:8000/`.
 
 If you have previously run a project on a different port, a `.foreman` file
 may be generated at the root of your directory. If so, make sure that this
-file is set to port `5000` or you will be unable to authenticate locally with MyUSA.
+file is set to port `8000` or you will be unable to authenticate locally with MyUSA.
```

Comment:
The local port used seems to be inconsistent throughout the project (some places still say `5000`, some `8000`)...shall we pick one?